### PR TITLE
docs: add security policy for openEquella

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [ ] screenshots are included showing significant UI changes
- [x] documentation is changed or added

##### Description of change

Adds security policy into .github/SECURITY.md, which GitHub automatically will associate with the security tab in the UI.
https://docs.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- A CVE issue is opened
+- A fix is created (ideally on a private fork)
+- On the embargo date:
+  - The fix is released
+  - The Advisory is published
+  - Notices are sent out on the [equella-users](https://groups.google.com/a/apereo.org/g/equella-users) and [equella-dev](https://groups.google.com/a/apereo.org/g/equella-dev) mail lists.
+
+The openEQUELLA Security Group is not responsible for fixing a given security issue. They are responsible to do the initial review, recommend a path forward, and guide the advisory to completion.
+
+The openEQUELLA Security Group generally focuses on the latest release for security issues, so as of Sep. 23rd, 2019, the focus would be on security issues in oEQ 2019.1.1.
```
Comment (cbeach47) [inline .github/SECURITY.md:21]: We should update the date / release:
`as of Sep. 23rd, 2019, the focus would be on security issues in oEQ 2019.1.1`

to be:
`as of Aug. 12th, 2020, the focus would be on security issues in oEQ 2020.1.3`

Comment (ChristianMurphy) [reply .github/SECURITY.md]: Thanks! updated

**Comment by ChristianMurphy** [comment]
Policy is now live at https://github.com/openequella/openEQUELLA/security/policy

**Comment by edalex-ian** [comment]
Good stuff! Thanks @ChristianMurphy :bowing_man:

**Pull request merged**